azure-pipelines.yml

```diff
@@ -26,10 +26,21 @@ steps:
   inputs:
     versionSpec: '10.16.2'
   displayName: use node 10.16.2
+  
+- task: CmdLine@2
+  inputs:
+    script: |
+      echo Update Chrome
```

Comment:
Same comment here as the action workflow

Reply:
Ditto, same response as above.